web/src/app/shared/store/classification-store/classification.state.ts

```diff
@@ -192,9 +185,18 @@ export class ClassificationState implements NgxsAfterBootstrap {
   }
 }
 
+function replaceClassifications(classifications: ClassificationSummary[], classification: Classification) {
```

Comment:
Honestly I hate this name.. Any suggestions?

Reply:
updateClassificationInList?

Reply:
nice naming :)